crates/gpui/src/platform.rs

```diff
@@ -61,9 +61,44 @@ pub(crate) fn current_platform() -> Rc<dyn Platform> {
 pub(crate) fn current_platform() -> Rc<dyn Platform> {
     Rc::new(LinuxPlatform::new())
 }
+// todo!("windows")
 #[cfg(target_os = "windows")]
 pub(crate) fn current_platform() -> Rc<dyn Platform> {
-    todo!("windows")
+    // Temporary ignore dead_code
```

Comment:
I think this dead code should be removed.

Reply:
I think it is actually used on other platforms, but removing it will result in a clippy error as dead_code for the symbols used here.
I think it would be better than going around with `#[allow(dead_code)]` on every symbol that causes an error, since this is where we will eventually make the change.

Reply:
I think it would be easier to remove this code and add:

```
// todo!("windows"): remove
#![cfg_attr(target_os = "windows", allow(dead_code))]
```

Reply:
Oh I see. There is similar code for linux

Reply:
I changed it